opentelemetry/proto/trace/v1/trace.proto

```diff
@@ -103,6 +103,12 @@ message Span {
   // field must be empty. The ID is an 8-byte array.
   bytes parent_span_id = 4;
 
+  // Flags from the W3C traceparent, if provided.
+  // There are two flags defined in the W3C Trace Context Level 2 spec:
```

Comment:
Maybe just have a link to the specification rather than call out the two values (in case of changes).

It may also be worth calling out this field may be empty / zero in older clients.

Reply:
Added 
```
// Note that Span flags were introduced in version 1.1 of the
// OpenTelemetry protocol.  Older Span producers do not set this
// field, consequently consumers should not rely on the absence of a
// particular flag bit to indicate the presence of a particular feature.
```